ui/src/components/Pin.tsx

```diff
@@ -62,7 +62,16 @@ export function Pin({
   const rightPaddingPoint = L.point(100, 400);
 
   return (
-    <Marker position={[story.latitude, story.longitude]} icon={icon}>
+    <Marker
+      position={[story.latitude, story.longitude]}
+      icon={icon}
+      onMouseOver={(e) => {
+        e.target.openPopup();
```

Comment:
What happens if someone has a marker open for marker A, what happens when you mouse over marker B?

Reply:
It will close marker A's preview since it left that target and open the preview for marker B